ansible/roles/nomad-client/templates/40-client.hcl

```diff
@@ -56,5 +56,12 @@ vault {
 plugin "docker" {
   config {
     extra_labels = ["*"]
+    # default from https://developer.hashicorp.com/nomad/docs/drivers/docker#allow_caps
+    allow_caps = [
```

Comment:
Consider using `to_json` here to format the list in a type-safe way.  Then you can just combine the default caps with the extra ones.